training/metrics.py

```diff
@@ -3,18 +3,15 @@
 from functools import reduce
 from tensorflow.python.util import object_identity
 
-
 def log10(x):
   base = 10.
   return tf.math.log(x) / tf.math.log(base)
 
-
 def calc_accuracy(y_true, y_pred):
   y_true = tf.math.argmax(tf.convert_to_tensor(y_true, tf.float32), axis=-1)
   y_pred = tf.math.argmax(tf.convert_to_tensor(y_pred, tf.float32), axis=-1)
   return tf.math.reduce_mean(tf.cast((tf.math.equal(y_true, y_pred)), dtype=tf.float32))
 
-
```

Comment:
Updated accordingly.

Reply:
@Net-Mist ,Happy that you wan to follow PEP8 ! Are there some differences for the google style you adopted ?

Coincidentally, I found this google style guide : https://google.github.io/styleguide/pyguide.html#3-python-style-rules which states 4 spaces for identation (like PEP8, while tensorflow is 2). Is there a statement on the style guide for tensorflow ?

Reply:
If I remember correctly we followed google style as most of the keras code we reuse are following spaces 2. It would be odd to use both conventions in the code. hence we stick with google style for spaces 2. 
I'm in for code readability and convenience which ever convention or a mix of both if that works.

Reply:
True, I made a double error here: I should have referred to tensorflow coding style and I meant **2 blank lines**, not spaces
Tensorflow coding style is defined here : https://www.tensorflow.org/community/contribute/code_style (so mainly PEP8 except for number of spaces). As @asbharath mention the main idea is to have consistent style between the engines, keras and our training codes. 
(@ClementPinard : we could define our own coding style and use tab, so every user can configure the size of the tabulation :p )